scripts/update-dependencies

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```

Comment:
for maintainers, I think you're missing a link to this script in the place they'd naturally discover it

Reply:
Oh wow we're somehow just completely missing a README? Oops. Added in 145c2436811286e89b35233171eb601a29b1babf.